### DISCUSSION TITLE
NextJS image optimization for Avatar component

### DISCUSSION THREAD

**Discussion by pulkitvyas08** [chakra-ui/chakra-ui#6639] [Q&A]
I found that the ChakraNextImage implementations of the Chakra UI image component and looking for something similar for the Avatar component that can:
1. Let me use the Charka UI Avatar props with NextJS image component optimizations
2. Fallbacks to the default Avatar icon if there is no valid source provided

Is such an implementation available in the community?

**Comment by alexomon018** [comment]
Hm I have managed to get a round this with a conditional.

` <chakra.span
                rounded={"full"}
                position="relative"
                overflow={"hidden"}
              >
                {obj.image ? (
                  <NextImage
                    src={obj.image}
                    alt={obj.name}
                    width={"100%"}
                    height={"100%"}
                    objectFit="cover"
                  />
                ) : (
                  <Avatar size="lg" name={obj.name} />
                )}
              </chakra.span>
`
